src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
Hmmm ... I like that third option but don't like the label name. That'll be a bit confusing as we have a totally separate semantic version of the ui. Maybe something like `pachyderm/dash:pachctl-1.3-latest` ?

That's a little goofy.